app/templates/components/exp-editor-frame.hbs

```diff
@@ -0,0 +1,15 @@
+<div {{action 'selectFrame' index}}>
+  <div class="row">
+    <div class="col-md-2">
+      <span class="index">
+      {{ plus-one index }}
+      </span>
+    </div>
+    <div class="col-md-10" style="white-space:normal">
```

Comment:
In general, I prefer putting style rules in CSS files.

There are quite a few places where the templates specify styles directly with the HTML tag, which could make it harder to maintain later. Moving them to CSS files could be beneficial.